app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -243,9 +223,13 @@ class BindableAdapter<T : Any> internal constructor(
       )
     }
 
+    /** Fragment injectable factory to create new [SingleTypeBuilder] */
     class Factory @Inject constructor(
       val fragment: Fragment
     ) {
+      /**
+       * Returns a new [SingleTypeBuilder] for the specified Data class type.
+       */
```

Comment:
```suggestion
      /** Returns a new [SingleTypeBuilder] for the specified Data class type. */
```

Can fit on one line. Ditto below.

Reply:
Noted.